### PR TITLE
Add indices count + indices names to all actions

### PR DESCRIPTION
**Pull request by shaharmor**
## Proposed Changes
  - Add the count of indices + indices names to all actions (Or at least the relevant ones) so its easier to get that info at the beginning of the action execution

### CODE REVIEW

File: curator/actions.py [hunk 1]
```diff
@@ -581,7 +581,7 @@ def do_action(self):
         self.index_list.filter_forceMerged(
             max_num_segments=self.max_num_segments)
         self.index_list.empty_list_check()
-        self.loggit.info('forceMerging selected indices')
+        self.loggit.info('forceMerging {0} selected indices: {1}'.format(len(self.index_list.indices), self.index_list.indices)
```
Comment (untergeek) [inline curator/actions.py]: This line is not properly terminated, so the CI tests will not even run.

File: curator/actions.py [hunk 2]
```diff
@@ -1019,7 +1019,7 @@ def do_action(self):
         seconds between retries.
         """
         self.snapshot_list.empty_list_check()
-        self.loggit.info('Deleting selected snapshots')
+        self.loggit.info('Deleting {0} selected snapshots: {1}'.format(len(self.snapshot_list.snapshots), self.snapshot_list.snapshots)
```
Comment (untergeek) [inline curator/actions.py]: This line is also missing a closing `)`

File: curator/actions.py (hunk 1 above)
Comment (untergeek) [reply curator/actions.py]: that is to say, it needs an additional `)`

Comment (shaharmor) [reply curator/actions.py]: Whoops, sorry. Edited on github 👎

**Pull request merged**